src/sampleData/activityData.js

```diff
@@ -125,7 +125,7 @@ const activityData = [
       "happy",
       "fearful",
       "angry",
-      "disguisted",
```

Comment:
lol thanks for fixing that bug!!

Reply:
😒😒😒😒😒😒😒😒😒😒😒😒😒😒😒😒😒😒😒😒😒😒😒😒😒😒😒😒😒

Reply:
aw Joe, don't cry, even I used it's instead of its in this PR description

Reply:
Good catch!!!